src/status_im/contexts/chat/home/view.cljs

```diff
@@ -114,13 +114,17 @@
                                              {:scroll-input (oops/oget % "nativeEvent.contentOffset.y")
                                               :shared-value scroll-shared-value})}])))
 
+(defn- on-new-message-press
+  []
+  (let [main-event [:show-bottom-sheet {:content chat.actions.view/new-chat}]]
+    (rf/dispatch [:profile/check-profile-update-prompt main-event])))
+
```

Comment:
Showing the sheet to update the profile when the user presses in the `+` button to send a message